sdv/metadata/table.py

```diff
@@ -84,6 +84,12 @@ class Table:
             The columns in the dataframe which are constant within each
             group/entity. These columns will be provided at sampling time
             (i.e. the samples will be conditioned on the context variables).
+        rounding (int, str or None):
+            Define rounding scheme for ``NumericalTransformer``.
```

Comment:
I would add an explanation of the possible values here (and on the other models)